com.musala.simple.students.spring/src/main/java/com/musala/simple/students/spring/StudentService.java

```diff
@@ -0,0 +1,85 @@
+package com.musala.simple.students.spring;
```

Comment:
We should think of adding unit tests for the service - I believe here we could easily make use of the Mockito's `when smt...then return...` functionality.